sdk/core/azure-core-tracing-opentelemetry/pom.xml

```diff
@@ -50,6 +50,18 @@
       <version>0.2.0</version> <!-- {x-version-update;io.opentelemetry:opentelemetry-sdk;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-test</artifactId>
+      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
```

Comment:
Given that tracing should be released in the same group as Azure Core should this take a dependency on the current version of Azure Core Test?